scripts/compute_branch_network.sh

```diff
@@ -15,6 +15,9 @@ if [ "${BRANCH}" = "rel/stable" ]; then
 elif [ "${BRANCH}" = "rel/beta" ]; then
     echo "betanet"
     exit 0
+elif [ "${BRANCH}" = "feature/alphanet" ]; then
```

Comment:
I'm increasingly concerned at the special casing we're doing to support this use case, especially as regards to future Feature Net work. We should keep track of this so we can revise the Feature Net design.